### PR TITLE
chore: fix typos

### PR DESCRIPTION
**Pull request by dundargoc**
Typos and similiarly small fixes will be collected here.

### CODE REVIEW

**Comment by 0xAdk** [comment]
there is a type-o in [doc/lua.txt:1046](https://github.com/neovim/neovim/blob/9fe8d2c9df2decccd09828cb99ae7a19635c7dc3/runtime/doc/lua.txt#L1046)
`but using lua tables` should be `by using lua tables`

**Comment by max397574** [comment]
I'm not sure if these docs are autogenerated
In treesitter.txt I found two typos:

`:h ts-predicate-any-of?`
should be
> any of the following arguments.
At least that^s how it's written for `contains?`

The other one is at `tsnode:end_()`
This is a trailing `_` in the name

**Comment by dundargoc** [comment]
@max397574: fixed. Thanks.

**Comment by simmsbra** [comment]
The help for the ternary operator is mistakenly tagged with the word "trinary" instead of "ternary" on [line 843 of doc/eval.txt](https://github.com/neovim/neovim/blob/8051fa1aff24952b9d077ba44bc2b84fd1626345/runtime/doc/eval.txt#L843).

[https://en.wikipedia.org/wiki/Ternary_operation#Computer_science](https://en.wikipedia.org/wiki/Ternary_operation#Computer_science)

FYI this looks to be the only instance of the word "trinary" in this repo ([searched using GitHub](https://github.com/neovim/neovim/search?q=trinary)).

Thank you!

**Comment by zeertzjq** [comment]
> The help for the ternary operator is mistakenly tagged with the word "trinary" instead of "ternary" on [line 843 of doc/eval.txt](https://github.com/neovim/neovim/blob/8051fa1aff24952b9d077ba44bc2b84fd1626345/runtime/doc/eval.txt#L843).
> 
> [en.wikipedia.org/wiki/Ternary_operation#Computer_science](https://en.wikipedia.org/wiki/Ternary_operation#Computer_science)
> 
> FYI this looks to be the only instance of the word "trinary" in this repo ([searched using GitHub](https://github.com/neovim/neovim/search?q=trinary)).
> 
> Thank you!

I think it's better to report this one to Vim.

File: runtime/doc/treesitter.txt [hunk 1]
```diff
@@ -128,7 +128,7 @@ tsnode:start()						*tsnode:start()*
 	Get the node's start position. Return three values: the row, column
 	and total byte count (all zero-based).
 
-tsnode:end_()						*tsnode:end_()*
+tsnode:end()						*tsnode:end()*
```
Comment (zeertzjq) [inline runtime/doc/treesitter.txt:131]: This is not a typo.

**Comment by zeertzjq** [comment]
> The other one is at tsnode:end_()
> This is a trailing _ in the name

That's not a typo.

**Pull request merged**